Src/Support/Google.Apis.Auth.Tests/OAuth2/Pkcs8Tests.cs

```diff
@@ -137,6 +137,10 @@ public void RsaFuzzTest()
                 Assert.Equal(publicKey.Modulus.ToByteArrayUnsigned(), Pkcs8.TrimLeadingZeroes(exportedParams.Modulus, false));
             }
         }
+#elif NETCOREAPP1_0 || NETCOREAPP1_1
+// RsaFuzzTest() can't run on net core.
```

Comment:
I've explicitly chosen not to run the test on netcoreapp, rather than just not running the test on anything that isn't net452 or net46.
As explained in another comment, I prefer to be very clear about this and cause compile errors; so we're forced to consider the right thing to do when adding new target frameworks.